src/components/metrics/card/name/index.js

```diff
@@ -1,108 +1,117 @@
 import React, {Component, PropTypes} from 'react'
 
-import TextArea from 'react-textarea-autosize'
+import {EditorState, Editor, ContentState, getDefaultKeyBinding, KeyBindingUtil} from 'draft-js'
 
 import './style.css'
 
+class SimpleEditor extends React.Component {
```

Comment:
It'd be nice if we could not name two separate components the same thing. Why not NameEditor?